### ISSUE TITLE
Failed to minify code while using react build script

### ISSUE THREAD

**Issue by rhrahul** [MetaMask/provider-engine#275]
this is the error I am getting while running `npm run build` in react

```
Creating an optimized production build...
Failed to compile.

Failed to minify the code from this file:

        ./node_modules/web3-provider-engine/node_modules/eth-block-tracker/dist/es5/index.js:10

```

Please assist me, what should I do ?
Thanks

**Comment by theone3nu** [comment]
for me also getting the same issue.

**Comment by AntonioJuliano** [comment]
Still happening for us as well

**Comment by kumavis** [comment]
error message is not helpful but sounds like this is uglify dying on >es5 code in the `dist/es5/index.js`

**Comment by kumavis** [comment]
thanks for report, fixed in eth-block-tracker@4.3.0
